### DISCUSSION TITLE
Firehose_Finder

### DISCUSSION THREAD

**Discussion by hoplik** [hoplik/Firehose-Finder#46] [General]
Исправлена ошибка установщика

<hr /><em>This discussion was created from the release <a href='https://github.com/hoplik/Firehose-Finder/releases/tag/24.8.5.0'>Firehose_Finder</a>.</em>

**Comment by alltern** [comment]
Hi can you please add loader of g996U or the s21 plus!

------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Привет, можете ли вы добавить загрузчик g996U или s21 плюс!

**Reply by hoplik** [reply]
Check the link below. This programmer may not suit for you. You need to check it on your device. Please note, that the antirollback version for this programmer is 00000008.
https://github.com/hoplik/Firehose-Finder/blob/master/fh_collection/4BF1A07B/8/Sams_G996U.elf

**Comment by alltern** [comment]
Okay thanks for telling me the anti rollback version I had no idea what it was before but I still appreciate the help so thanks! just wish there was a SW REV C firehose loader. but samsung is just a scummy company adding rollback versions for programmers.

**Reply by hoplik** [reply]
Sorry, bro. There is no other way.

**Reply by alltern** [reply]
ahh ok just going to have to wait probably months. so unfortunate samsung is  a scumbag company. Well good news is in the meantime im getting a google pixel and the reason i wanted the firehose was so i can install a hardened OS. so ill just do that on the google pixel.

**Comment by alltern** [comment]
Also even if it didnt work out in the end I applaud you and @bkerler for still trying your best to provide these loaders without charging a huge price on it with no garuntee it will work. I respect that.

**Reply by hoplik** [reply]
Yes, Bjorn is mega. Looking at his project, I decided to write my own alternative one.
